src/NServiceBus.AcceptanceTests/Sagas/When_correlating_on_null.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    class When_correlating_on_null : NServiceBusAcceptanceTest
```

Comment:
```suggestion
    class When_correlation_property_is_null : NServiceBusAcceptanceTest
```

Reply:
👍 Will change manually as the file has to be renamed as well.